packages/convertkit/src/forms/subscribe/index.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react'
+import * as Yup from 'yup'
+import axios from 'axios'
+import queryString from 'query-string'
+import {useFormik} from 'formik'
+import {useRouter} from 'next/router'
+import {CK_SUBSCRIBER_KEY} from '@skillrecordings/config'
+import {Input, Button} from '@skillrecordings/react/dist/components'
+
+type SubscribeFormOptions = {
+  /**
+   * The content of submit button.
+   * @type string
+   */
+  actionLabel?: string
+  /**
+   * The URL to redirect to after succesfully submitting a form. Default is `/confirm`.
+   * Submitted `email_address` will be automatically passed as URL param.
+   * Providing `false` value will skip the redirect entirely.
+   * @type string | false
+   */
+  onSuccessRedirectUrl?: string | false
+}
+
+type SubscribeFormProps =
+  | ({
+      /**
+       * The tag id to tag a subscriber with.
+       * @type number
+       */
+      tag: number
+      form?: never
+      sequence?: never
+    } & SubscribeFormOptions)
+  | ({
+      tag?: never
+      /**
+       * The form id to subscribe to. Default is `process.env.NEXT_PUBLIC_CONVERTKIT_SIGNUP_FORM`.
+       * @type number
+       */
+      form: number
+      sequence?: never
+    } & SubscribeFormOptions)
+  | ({
+      tag?: never
+      form?: never
+      /**
+       * Sequence ID to subscribe to.
+       * @type number
+       */
+      sequence: number
+    } & SubscribeFormOptions)
+  | ({tag?: never; form?: never; sequence?: never} & SubscribeFormOptions)
```

Comment:
Since we're picking an endpoint based on whether `tag`, `form` or `sequence` id is provided, we can allow for **only providing one of them**. Or none, in which case we fallback to default setting.

Reply:
convertkit let’s you send an arbitrary number of tags to any of these endpoints. we use that on the backend, but haven’t on the front end. it will make the logic more challenging

Reply:
True that. Added this to README under improvements.